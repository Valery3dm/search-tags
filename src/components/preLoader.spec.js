import React from 'react';
import Enzyme, { shallow } from 'enzyme';
import Adapter from '@wojtekmaj/enzyme-adapter-react-17';

import PreLoader from './preLoader';

Enzyme.configure({ adapter: new Adapter() });


describe("cards component", () => {

    it("should render cards component", () => {
        const component = shallow(<PreLoader />);
        expect(component).toMatchSnapshot();
    })
})