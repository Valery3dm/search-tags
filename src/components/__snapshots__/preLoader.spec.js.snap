// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`cards component should render cards component 1`] = `ShallowWrapper {}`;
